gen/definitions/device_vtep_policy.yaml

```diff
@@ -0,0 +1,110 @@
+---
+name: Device VTEP Policy
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/devices/devicerecords/%v/vteppolicies
+res_description: >-
+  This resource can manage the Device VTEP Policy. Practicioners should ensure only one resource
+  `fmc_device_vtep_policy` exists for a single `fmc_device`, because the FMC API responds with the same singe UUID for
+  every request to create a new VTEP Policy on the same Device.
+  Thus multiple resources per Device would unexpectedly overwrite the same set of settings.
+doc_category: Devices
+test_tags: [TF_VAR_device_id]
+attributes:
+  - tf_name: device_id
+    type: String
+    reference: true
+    requires_replace: true
+    description: >-
+      UUID of the parent device (fmc_device.example.id).
+    example: 76d24097-41c4-4558-a4d0-a8c07ac08470
+    test_value: fmc_device_physical_interface.test.device_id
+  - model_name: nveEnable
+    tf_name: nve_enabled
+    description: Indicates whether to enable NVE on the `device_id`. Can only be false if `vteps` are empty.
+    type: Bool
+    default_value: "true"
+    exclude_example: true
+    minimum_test_value: "false"
+    test_value: "true"
+  - model_name: vtepEntries
+    tf_name: vteps
+    type: List
+    description: List that can either be empty or contain one VTEP object.
+    attributes:
+      - model_name: id
+        data_path: [sourceInterface]
+        tf_name: source_interface_id
+        id: true
+        type: String
+        description: >-
+          UUID of the source interface (e.g. fmc_physical_interface.example.id). It cannot refer to a subinterface.
+        example: 76d24097-41c4-4558-a4d0-a8c07ac08470
+        test_value: fmc_device_physical_interface.test.id
+      - model_name: nveVtepId
```

Comment:
Should we hardcode it for now? In case we want to make it configurable later, we can always introduce an optional attribute.

Reply:
I decided to keep it here because:

1. Visible in UI
2. The corresponding attribute of vni cannot be hardcoded, because even today the `null` for it is a meaningful choice (described in UI as "this VNI interface is not mapped to VTEP"). I mean this place: https://github.com/netascode/terraform-provider-fmc/pull/108/files#diff-facc7a33357b6f494add4e0f53ff9951a11aec14c06daf4711d75fe9f2b50c52R47

Please confirm, do you want to remove vni nve_number as well and put some bool in its place?

Reply:
If it can be either "null" or "1", then let's keep it as is.